vtex/omnishipping
# Fixes showing address list when there is only one address

#### What is the purpose of this pull request?
Fixes showing address list when there is only one address

#### How should this be manually tested?
1. Add [items to cart](https://fernando--vtexgame1.myvtex.com/checkout/cart/add?&workspace=fernando&sku=298&qty=1&seller=1&sc=1)
2. Proceed to shipping and type `22071060`
3. Address list should not be shown

#### Screenshots or example usage
|Before|After|
|-|-|
|![Screen Shot 2019-10-16 at 12 14 57](https://user-images.githubusercontent.com/5608421/66932893-acc04180-f00e-11e9-9fa7-aea1e37d89cf.png)|![Screen Shot 2019-10-16 at 12 15 21](https://user-images.githubusercontent.com/5608421/66932903-afbb3200-f00e-11e9-9f10-a55cbf6dad18.png)|

#### Types of changes
